readthedocs_ext/readthedocs.py

```diff
@@ -70,7 +71,18 @@ def update_body(app, pagename, templatename, context, doctree):
         # Only insert on our HTML builds
         return
 
-    if theme_css not in app.builder.css_files:
+    inject_css = True
+
+    # After v0.3.0 of the sphinx theme, the theme CSS should not be injected
+    # This decouples the theme CSS (which is versioned independently) from readthedocs.org
+    if theme_css.endswith('sphinx_rtd_theme.css'):
+        try:
+            import sphinx_rtd_theme
+            inject_css = LooseVersion(sphinx_rtd_theme.__version__) <= LooseVersion('0.3.0')
```

Comment:
Do we need to update the theme version here, or will this work with 0.3.0 and 0.3.1, and this logic can stay?

Reply:
Believe it depends on https://github.com/rtfd/sphinx_rtd_theme/pull/614 being released right?

Reply:
> Believe it depends on rtfd/sphinx_rtd_theme#614 being released right?

Correct so perhaps we should bump to 0.4.0.